designer/package.json

```diff
@@ -60,6 +60,8 @@
     "lodash": "^4.17.21",
     "moment-timezone": "^0.5.31",
     "nanoid": "^3.1.12",
+    "pa11y": "^6.1.1",
+    "pa11y-ci": "^3.0.1",
```

Comment:
these should be defined as devDependencies, since they're not used in the production builds. It seems like it is being run via `dlx`/`npx` however, which is a way of running packages without defining them in package.json. 

I'm ok with running dlx/npx on the CI runner, or installing it as a dev dependency, but ideally we shouldn't mix approaches